yt/data_objects/selection_objects/data_selection_objects.py

```diff
@@ -682,8 +682,8 @@ def cut_region(self, field_cuts, field_parameters=None, locals=None):
 
         >>> ds = yt.load("RedshiftOutput0005")
         >>> ad = ds.all_data()
-        >>> cr = ad.cut_region(["obj['gas', 'temperature'] > 1e6"])
-        >>> print(cr.quantities.total_quantity(("gas", "cell_mass")).in_units("Msun"))
+        >>> cr = ad.cut_region(["obj[('gas', 'temperature')] > 1e6"])
+        >>> print(cr.quantities.total_quantity(("gas", "cell_mass")).in_units('Msun'))
```

Comment:
looks like black will overwrite this in parts

Reply:
Damn, I accepted the wrong side of the merge...